Commands/general/verify.ts

```diff
@@ -0,0 +1,56 @@
+import { SlashCommandBuilder, EmbedBuilder, ChatInputCommandInteraction, User, Colors } from 'discord.js';
+import { DiscordClient } from '../../Utils/DiscordClient';
+import fs from 'fs';
+import { whisperChat } from '../../Utils/IngameChat';
+export const data = new SlashCommandBuilder()
+    .setName('verify')
+    .setDescription('Yerify yourself with the ingame bot')
+    .addStringOption((o) => o.setName(`code`).setDescription(`Put the code you recieved here.`).setRequired(true))
+
+export function execute(interaction: ChatInputCommandInteraction, client: DiscordClient) {
+        const code = interaction.options.getString("code") || "null"
+        let pending = fs.readFileSync(__dirname + "/../../Verification/pendingVerification.json","utf-8");
+        let done = fs.readFileSync(__dirname + "/../../Verification/doneVerification.json","utf-8");
+        var pendingData = JSON.parse(pending);
+        var doneData = JSON.parse(done)
+        var user = `${interaction.user.username}#${interaction.user.discriminator}`
```

Comment:
All of these could be constants except for pendingData but I will talk about a potential solution to that. Additionally instead of making more separate variables you could just do `const pending = JSON.parse(fs.readFileSync(__dirname + "/../../Verification/pendingVerification.json", "utf-8"));` or if it's too long for your liking you could just make a function in the Utils folder that reads & parses it that way it is shorter.

Reply:
I will probably just change this and...